dashboard/app/models/pd/application/principal_approval_application.rb

```diff
@@ -42,12 +42,17 @@ class PrincipalApprovalApplication < ApplicationBase
 
     belongs_to :teacher_application, class_name: 'Pd::Application::TeacherApplication',
                primary_key: :application_guid, foreign_key: :application_guid
+    after_create :update_teacher_app_status
 
     # @return a valid year (see Pd::SharedApplicationConstants::APPLICATION_YEARS)
     def year
       application_year
     end
 
+    def update_teacher_app_status
+      teacher_application.update!(status: 'unreviewed') if teacher_application.status == 'awaiting_admin_approval'
```

Comment:
Are these statuses defined as constants anywhere? If not, that might be a nice clean up after launch.

Reply:
Good call! I'll add it to the refactor list. I'm thinking I'll get good test coverage and then delve into refactors